### ISSUE TITLE
dissolutionLocation 

### ISSUE THREAD

**Issue by VladimirAlexiev** [schemaorg/schemaorg#302]
Schema has properties foundationLocation and dissolutionDate but no dissolutionLocation. In contrast, for a person all 4 properties are present (birth/death place/date).

Getty ULAN has such a field, and 70 records that use it.

Often the foundation place and dissolution place are the same place. There are instances, however, where the places are different. For instance, during WWII a firm founded in Berlin may have been forced to move to Paris, and then was eventually dissolved in Paris.

Getty internal tracker: https://jira.getty.edu/browse/ITSLOD-283

**Comment by danbri** [comment]
Thanks, noted. We're quite backlogged but I've put this on the someday pile. We have some related vocabulary also in https://www.w3.org/wiki/WebSchemas/HistoricalDataSchema that was never finally incorporated.

**Comment by nichtich** [comment]
https://schema.org/location should also be super-property of https://schema.org/birthPlace and https://schema.org/deathPlace

**Comment by VladimirAlexiev** [comment]
`location` should also be super-prop of `foundationLocation, dissolutionLocation`

**Comment by danbri** [comment]
Good point @VladimirAlexiev - would you care to send a PR to add dissolutionLocation via pending? and another to add those super-property assertions?

> Getty ULAN has such a field, and 70 records that use it.

~~70~~ 79 records! 📈

Are there other usecases, or applications consuming the data? What does Wikidata do in this area?

If the primary scenario here is cultural heritage, should the definition indicate that a typical value is most likely to be a city rather than a country? Countries / states can get restructured and partitioned, whereas cities tend to be more enduring entities even if they get renamed. If the usecase is closer to law, e-government etc., perhaps there are other requirements? Are the Getty records mostly cities?

Ok, I tried [tweaking](http://vocab.getty.edu/sparql?query=select%20*%20%7B%0A%20%20%3Fx%20schema%3AdissolutionLocation%20%3Fy%20.%0A%20OPTIONAL%20%7B%20%3Fy%20%3Fsomeprop%20%3Fsomevalue%20.%20%7D%20%0A%7D&toc=Finding_Subjects&implicit=true&equivalent=false&_form=/queriesF) your Getty SPARQL to look, but from a quick check it seems they're very specific point locations rather than e.g. cities, named settlements or whatever.

```
select * {
  ?x schema:dissolutionLocation ?y .
 OPTIONAL { ?y ?someprop ?somevalue . } 
}
```

**Comment by VladimirAlexiev** [comment]
@danbri Can you please point me to a similar PR?

[They are cities](http://vocab.getty.edu/sparql?query=select%20*%20%7B%0A%20%20%3Fbio%20schema%3AdissolutionLocation%20%3Fplace.%0A%20%20%3FagentConcept%20foaf%3Afocus%2Fgvp%3AbiographyPreferred%20%3Fbio%3B%20gvp%3AprefLabelGVP%2Fxl%3AliteralForm%20%3FagentName.%0A%20%20%3FplaceConcept%20foaf%3Afocus%20%3Fplace%3B%20gvp%3AprefLabelGVP%2Fxl%3AliteralForm%20%3FplaceName.%0A%7D&toc=Finding_Subjects&implicit=true&equivalent=false&_form=/queriesF) but the Getty representation is a bit more involved so the query is more complex:
```sparql
select * {
  ?bio schema:dissolutionLocation ?place.
  ?agentConcept foaf:focus/gvp:biographyPreferred ?bio; gvp:prefLabelGVP/xl:literalForm ?agentName.
  ?placeConcept foaf:focus ?place; gvp:prefLabelGVP/xl:literalForm ?placeName.
}
```

I think datasets will expose whatever places they have: of course a more specific place like city is better, but if they only have data at the granularity of country, that's also ok.

Wikidata has "inception date", [location of formation](https://www.wikidata.org/wiki/Property:P740), [dissolution date](https://www.wikidata.org/wiki/Property:P576), "birth place", "death place" but no "dissolution location". Eg see the [Beatles](https://www.wikidata.org/wiki/Q1299).

**Comment by gmackenz** [comment]
Closing for now as there has been no updates in the past 4 years. Will reopen if there is need and a more importantly a github pull request.
